src/ReinforcementLearningCore/src/policies/agent/agent_srt_cache.jl

```diff
@@ -1,65 +1,49 @@
 import Base.push!
 
-mutable struct SRT{S,R,T}
+mutable struct SA{S,A}
```

Comment:
Why is there now the action in it? I would let it like it is. Can you describe why you think that this is necessary? In my opinion, the reward and terminal information is necessary.

Reply:
Before anything is merged about the cache part and the handling of optimise, it should be thought of how the optimization may be done:

- Policy may be updated after each step agent takes
- Policy may only be updated after environment terminated
- The two above with different update semantics